sql/rowexec/join_iters.go

```diff
@@ -520,7 +525,7 @@ func (i *fullJoinIter) Next(ctx *sql.Context) (sql.Row, error) {
 			continue
 		}
 		// (null, right) only if we haven't matched right
```

Comment:
Is this comment still correct? It says `null` but you've replaced the `nil` in the LoC below.